deku-derive/src/macros/deku_read.rs

```diff
@@ -128,7 +128,6 @@ fn emit_struct(input: &DekuData) -> Result<TokenStream, syn::Error> {
         });
     }
 
-    // println!("{}", tokens.to_string());
```

Comment:
The comment print can stay :) it's useful for debugging the output when cargo expand fails. I can add a comment to it.